package.json

```
@@ -0,0 +1,111 @@
+{
+  "name": "sheikah",
+  "productName": "Sheikah",
+  "version": "0.1.0",
+  "description": "A Witnet compatible desktop wallet and data requests development environment.",
+  "main": "src/electron-starter.js",
+  "scripts": {
+    "build": "react-scripts build",
+    "dev": "npm-run-all --parallel start electron-dev",
+    "electron-dev": "sleep 2 & ELECTRON_START_URL=http://localhost:3000 electron .",
+    "electron": "electron .",
+    "format-css": "npx prettier --write src/**/*.css --parser=css",
+    "format-js": "npx prettier --write {src/**/*.{js,jsx},stories/**/*.js} --config .prettierrc",
+    "format": "yarn format-js && yarn format-css",
+    "lint-css": "npx stylelint ./**/*.css --config .stylelintrc",
+    "lint-js": "npx eslint {src/**/*.{js,jsx},stories/**/*.js} -c .eslintrc",
+    "lint": "yarn lint-js && yarn lint-css",
+    "start": "react-scripts start",
+    "storybook": "start-storybook -p 6006",
+    "test": "react-scripts test",
+    "travis": "yarn lint && yarn fmt!"
```

Let's take the chance to rename this for the sake of independence from a particular CI provider and for uniformity with witnet-rust:
```suggestion
    "ci": "yarn lint && yarn fmt!"
```